packages/sstimap/PKGBUILD

```diff
@@ -0,0 +1,44 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=sstimap
+_pkgname=SSTImap
+pkgver=12.0d45b44
+pkgrel=1
+pkgdesc='Automatic SSTI detection tool with interactive interface.'
+arch=('any')
+groups=('blackarch' 'blackarch-webapp' 'blackarch-exploitation')
+url='URL pointing to download section of package'
+license=('gpl3')
+depends=('python')
+makedepends=('git' 'python-setuptools' 'python-pip')
+source=("git+https://github.com/vladko312/$_pkgname.git")
+sha512sums=('SKIP')
+install="$pkgname.install"
+
+pkgver() {
+  cd $_pkgname
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
```

Comment:
```suggestion
pkgver() {
  cd $_pkgbase

  ( set -o pipefail
    git describe --long --tags --abbrev=7 2>/dev/null |
      sed 's/\([^-]*-g\)/r\1/;s/-/./g' ||
    printf "%s.%s" "$(git rev-list --count HEAD)" \
      "$(git rev-parse --short=7 HEAD)"
  )
}
```

Reply:
Is this something new we are doing it now ?  This is not in the pkgbuild scripts. let me know. Thanks.

Reply:
[Here](https://github.com/BlackArch/blackarch-pkgbuilds/blob/b8fd779ec987a8d29f543a5cc5b729074ece638a/PKGBUILD-generic-git#L17) the reference. I think it should be extended to all PKGBUILD templates dealing with git: https://github.com/BlackArch/blackarch-pkgbuilds/pull/24